resources/lib/bom/bom_radar.py

```
@@ -112,26 +114,42 @@ def download_background(radar_code, file_name, path):
-
+    if not os.path.isfile(os.path.join(path, out_file_name)):
+        Logger.debug(f"Downloading missing background image....[{file_name}] as [{out_file_name}]")
+
+        # Try bundled national background first (if available)
+        if file_name == 'IDR00004.background.png' and CWD:
+            src = os.path.join(CWD, "resources", "IDR00004.background.png")
+            if os.path.isfile(src):
+                Logger.debug("Copying bundled national radar background")
+                try:
+                    shutil.copy(src, os.path.join(path, "background.png"))
+                    return
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)